src/app/main.scss

```diff
@@ -221,9 +221,6 @@ input[type='search'] {
   padding-right: constant(safe-area-inset-right);
   padding-right: env(safe-area-inset-right);
   scroll-margin-top: $header-height;
-  .gradientBackground & {
-    padding-top: 12px;
```

Comment:
This was causing some scrolling issues in the header on beta. Because the orange line was moved up, the header ended up getting some more padding. Can revisit, but wanted to fix for now.